sample-notificator-app/src/main/groovy/life/qbic/samplenotificator/components/CreateNotificationPresenter.groovy

```diff
@@ -12,29 +11,25 @@ import life.qbic.business.subscription.fetch.FetchSubscriberOutput
  * @since: 1.0.0
  *
  */
-class CreateNotificationPresenter implements CreateNotificationOutput, FetchSubscriberOutput {
+class CreateNotificationPresenter implements CreateNotificationOutput {
 
-    Map<Subscriber, String> notificationPerSubscriber
+    List<NotificationContent> notifications
```

Comment:
why is this a class variable? I think this should be like this because you dont have any advantage (or do I miss it?) to store it. It should only be used within the context of the createNotification() method :)